client/app/app.js

```diff
@@ -31,12 +31,12 @@ angular.module('impactApp', [
   })
   .run(function($rootScope, $window, $location) {
     $rootScope.$on('$stateChangeSuccess', function(event, toState) {
-      window.scrollTo(0, 0);
+      $window.scrollTo(0, 0);
 
       if (toState.data && toState.data.title) {
-        $rootScope.title = toState.data.title + ' - Votre MDPH en ligne';
+        $window.document.title = toState.data.title + ' - Votre MDPH en ligne';
       } else {
-        $rootScope.title = 'Votre MDPH en ligne';
+        $window.document.title = 'Votre MDPH en ligne';
```

Comment:
Ici j'utilise $window.document.title plutôt qu'un binding angular sur la balise `<title>` car:
 - c'est légèrement plus rapide (http://stackoverflow.com/questions/15790432/placement-of-the-ng-app-directive-html-vs-body)
 - pa11y sortait une erreur `head > title cannot be empty`, probablement car le binding n'était pas encore fait au moment du test